src/content/go/database_sql/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "database/sql"
+date: 2022-12-02T12:56:00+05:30
+draft: false
+weight: 1
+tags: ["go", "sql"]
+---
+
+SQLCommenter provides drop-in replacement for [go's `database/sql`](https://pkg.go.dev/database/sql) library. You can start using the [SQLcommenter's `database/sql`](https://github.com/google/sqlcommenter/tree/master/go/database/sql) without changing your business logic code and use one of the frmaework instrumentation to proapagate tags and traceparent to the SQL queries.
+
+## Installation
+
+```shell
+go get -u github.com/google/sqlcommenter/go/database/sql
+```
+
+## Usage
+
+Please use the sqlcommenter's default go-sql database driver to execute statements. 
```

Comment:
Should we mention that connections should be created using sqlcommenter's `sql::Open()` method here ?